thoth-api/src/account/model.rs

```diff
@@ -63,12 +63,29 @@ pub struct NewPublisherAccount {
     pub is_admin: bool,
 }
 
+#[derive(Debug, Clone, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
```

Comment:
What's the function of this attribute (and the one below)? Similar structs in other modules don't use it.

Reply:
It ensures that the attributes are output camel-cased. The JSON output by juniper does this by default so there's no need to include it (though we could). We could also include it in `Token`, but all the attributes are single-worded.